src/main/java/core/basesyntax/Storage.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Storage {
+    private List<Bet> betList = new ArrayList<>();
+    private List<User> userList = new ArrayList<>();
+
+    public List<User> getUserList() {
+        return userList;
+    }
+
+    public List<Bet> getBetList() {
+        return betList;
+    }
```

Comment:
these methods will also be static
or you can give direct access to your lists from outside

Reply:
done